Makefile

```diff
@@ -1407,6 +1407,28 @@ lint-py:
 	$(warning Run 'make lint-py-build')
 endif
 
+.PHONY: lint-yaml-build
+# python -m pip install yamllint
+# Try with '--system' if it fails without; the system may have set '--user'
+# Jenkins CI lint job uses python2 for linting. When Node.js 12.x is EOL, we
+# can remove that from the Jenkins job config and $(PYTHON) for YAML linting
+# just like everywhere else. (yamllint does not support Python 2.)
+lint-yaml-build:
+	$(info Pip installing yamllint on $(shell $(PYTHON) --version)...)
+	python3 -m pip install --upgrade -t tools/pip/site-packages yamllint || \
+		python3 -m pip install --upgrade --system -t tools/pip/site-packages yamllint
```

Comment:
```suggestion
	$(PYTHON) -m pip install --upgrade -t tools/pip/site-packages yamllint || \
		$(PYTHON) -m pip install --upgrade --system -t tools/pip/site-packages yamllint
```

Reply:
> I think this should be using the env variable.

It can't because the Jenkins CI job overrides the variable. See the comment starting on line 1413. Once 12.x is EOL, we can update the Jenkins job and make the change you suggest here. Alternatively, we could modify the Jenkins job right now, but I'd be inclined to land this first, *then* mess with it, *then* change this to the suggestion above. Trying to get a Jenkins config change integrated into this is too many moving parts in different repos for my taste.